apps/atomy-q/API/.env.example

```
@@ -53,6 +53,20 @@ ATOMY_MIN_VENDORS_PREVIEW=1
 ATOMY_CONFIDENCE_THRESHOLD=0.7
 ATOMY_SEED_TENANT_ID=
 
+# Quote intelligence
+# `deterministic` is the alpha default. Set `llm` only when the provider contract below is fully configured.
+QUOTE_INTELLIGENCE_MODE=deterministic
+# Required only when `QUOTE_INTELLIGENCE_MODE=llm` (for example: openai).
+QUOTE_INTELLIGENCE_LLM_PROVIDER=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The QUOTE_INTELLIGENCE_LLM_PROVIDER key should go before the QUOTE_INTELLIGENCE_MODE key

(UnorderedKey)